protocol/contracts/admin/VestingPayments.sol

```diff
@@ -86,7 +83,14 @@ contract VestingPayments is IVestingPayments, PaymentBase {
         /// @dev No need to check if `_schedule.revoked` because id is removed from _activeVestingIds so it will fail `isActiveVestingId`
 
         uint256 vestedNow = _getTotalVestedAtCurrentTime(_schedule);
-        revokedAccountsReleasable[_schedule.recipient] = vestedNow;
+        uint256 unvested = _schedule.amount - vestedNow;
+        if (unvested > 0) {
+            totalVestedAndUnclaimed -= unvested;
+        }
+        // Claimable delta at revoke time
+        uint256 releasableNow = vestedNow - _schedule.distributed;
+        // Persist per-schedule claimable checkpoint
+        _schedule.revokedReleasable = releasableNow.encodeUInt128();
```

Comment:
Was a specific test case added to check multiple revoke's for the same user? It doesn't look like it, but that's the exact case this was intended to fix. So that should 💯 be added in (unless i just missed it in the existing tests)